src/checkers/english.rs

```diff
@@ -97,7 +104,8 @@ fn normalise_string(input: &str) -> String {
     // TODO add more puncuation
     input.to_lowercase().replace(
         &[
-            '(', ')', '!', '/', ',', '?', '\"', '.', ';', ':', '\'', '`', ';', ':', '~', '^',
+            '(', ')', '!', '/', ',', '?', '\"', '.', ';', ':', '\'', '`', ';', ':', '~', '^', '#',
```

Comment:
you can use [is_ascii_punctuation](https://doc.rust-lang.org/std/primitive.char.html#method.is_ascii_punctuation) .

Something like this:
```rust
input.chars().filter(|x| !x.is_ascii_punctuation()).collect()
```